contrib/docker-compose/standalone/mkconfig.bash

```diff
@@ -3,10 +3,10 @@
 HERE=$(cd $(dirname $0); pwd)
 set -eux
 
-[ -e "$HERE/site.profile" ]
-. $HERE/site.profile
+[ -e "$HERE/.env" ]
+. "$HERE/.env"
 
-sed 's/POSTGRES_USER/'$POSTGRES_USER'/;s/POSTGRES_PASSWORD/'$POSTGRES_PASSWORD'/;' \
-	$HERE/hockeypuck/etc/hockeypuck.conf.tmpl > $HERE/hockeypuck/etc/hockeypuck.conf
-sed 's/FQDN/'$FQDN'/' \
-	$HERE/nginx/conf.d/hockeypuck.conf.tmpl > $HERE/nginx/conf.d/hockeypuck.conf
+sed 's/POSTGRES_USER/'"$POSTGRES_USER"'/;s/POSTGRES_PASSWORD/'"$POSTGRES_PASSWORD"'/;s/RELEASE/'"$RELEASE"'/;s/FQDN/'"$FQDN"'/;s/FINGERPRINT/'"$FINGERPRINT"'/;' \
```

Comment:
Recently I've found [envsubst](https://command-not-found.com/envsubst) very nice for this sort of thing.

Reply:
Nice! I've been looking for something like this for a while - tried various preprocessors but they were too heavy. I'll do this in a future PR if that's OK...